From 88b2f61bef4799dc64f7cc1ca8169f90be04816b Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 09 Mar 2021 19:28:14 +0200
Subject: [PATCH] [SW-34382] preboot: gaudi: increase sram

Handling boot info from scratchpad sram, specifically nvme
descriptor, has increased the bss utilization.

Change-Id: If3048a1f21769ee010363d0cd7e88eb72da0aff7
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index 360cf3d..684969e 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -8,7 +8,7 @@
 #define COMMON_GAUDI_H_
 
 #define PPBOOT_SIZE			0x4000		/* 16KB */
-#define PREBOOT_SIZE			0x12000		/* 72KB */
+#define PREBOOT_SIZE			0x14000		/* 80KB */
 #define SRAM_SIZE			0x1400000	/* 20MB */
 
 #define HL_SP_SRAM_BASE			0x7FFBFE0000
